src/components/PetsList.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import FilterList from "./FilterList";
+import PetCard from "./PetCard";
+import usePets from "../hooks/usePets";
+import { PetItems } from "../files/PetsData";
+import { Alert } from "@baltimorecounty/dotgov-components";
+
+const PetsList = () => {
+  //const [{ hasError, isLoading }] = usePets();
+
+  // if (hasError) {
+  //   return (
+  //     <Alert className="status" type="error">
+  //       <p>
+  //         Unable to retrieve the list of Baltimore County adoptable pets. Please
+  //         try again in a couple of minutes.
+  //       </p>
+  //     </Alert>
+  //   );
+  // }
+
+  return (
+    <React.Fragment>
+      {/* {isLoading ? (
+        <p>Loading Baltimore County Adoptable Pets...</p>
+      ) : ( */}
+      <div className="row">
+        <FilterList
+          items={PetItems}
+          renderItem={props => (
+            <div className="d-flex col-12" key={props.id}>
```

Comment:
there are some layout issues. we can remove the bootstrap from here

```js
renderItem={props => <PetCard key={props.id} {...props} />}
```

then to wrap around the filter list, we would want 
```jsx
<div class="container">
      <div class="row">
        <div className="col">
...
```

This will ensure hte proper spacing on mobile.

Reply:
done